applications/app_balance.c

```diff
@@ -419,13 +423,18 @@ static void calculate_setpoint_interpolated(void){
 }
 
 static void apply_constanttilt(void){
```

Comment:
Since you are now doing 2 kinds of adjustment in this function perhaps we can rename it to apply_noseangling or something like that.

Same for the float constanttilt_target variable as it's now the sum of the 2 tilts.

Reply:
I was thinking that too, but didn't want to rename your stuff without your input though. Will do